provisioning_client/devdoc/azure_provisioning_faq.md

```diff
@@ -0,0 +1,39 @@
+# Azure Iot Provisioning Client FAQ
+
+Here is where we shall try to answer question pertaining to the IoT Provisioning Client.  Please read the following documents for an overview of the DPS service.
+
+## x509 Provisioning
+
+- How does Provisining match the x509 certificates to an enrollment?
```

Comment:
I'm going to be a grammar snob and edit your grammar all over, sorry in advance.

Should be: How does the provisioning service match the device's X.509 certificate to an enrollment?